runtime/rasdump/dmpsup.c

```diff
@@ -577,7 +577,42 @@ configureDumpAgents(J9JavaVM *vm)
 		}
 		if ( IS_CONSUMABLE(j9vm_args, xdumpIndex) && !IS_CONSUMED(j9vm_args, xdumpIndex) )
 		{
-			GET_OPTION_VALUE(xdumpIndex, ':', &optionString);
+			BOOLEAN isMappedToolDump = FALSE;
+			/* Handle mapped tool dump options */
+			if (HAS_MAPPING(j9vm_args, xdumpIndex)) {
+				char *mappingJ9Name = MAPPING_J9NAME(j9vm_args, xdumpIndex);
```

Comment:
Does it work to set VMOPT_XDUMP_TOOL_OUTOFMEMORYERROR_EXEC_EQUALS  to `"events=systhrow,filter=java/lang/OutOfMemoryError,exec="` and just omit the `-Xdump:tool:`?

Reply:
We'd still need to know it's a tool mapping, but perhaps this can be added to the mapping definition.

Reply:
> Does it work to set VMOPT_XDUMP_TOOL_OUTOFMEMORYERROR_EXEC_EQUALS to `"events=systhrow,filter=java/lang/OutOfMemoryError,exec="` and just omit the `-Xdump:tool:`?

I think that I might need to keep `-Xdump:tool:` in the string, otherwise the option may not be picked up as a dump option in `dmpsup.c`, since `configureDumpAgents()` looks for arguments in `vmArgsArray` that contain `VMOPT_XDUMP` ("-Xdump") and consumes them accordingly.